tests/api/features/ServiceMonitoring.feature

```diff
@@ -0,0 +1,33 @@
+Feature:
+    In order to monitor services
+    As a user
+    I want to get service information using api
+
+    Background:
+        Given a running instance of Centreon Web API
+
+    Scenario: Browse Services
```

Comment:
As details and list are related to browsing, could you change the scenario title ?